src/styles/_catalog.scss

```
@@ -19,4 +21,25 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  @media (max-width: 1023px) {
+    .catalog {
+      grid-template-columns: repeat(3, $columns-width);
+      margin: 0 auto;
+    }
+  }
```

```suggestion
  @media (max-width: 1023px) {
    grid-template-columns: repeat(3, $columns-width);
    margin: 0 auto;
  }
```
no need to add selector, when media is inside scss block, now it does not work

---

Thanks, fixed it